packages/rule-no-p3p/tests/tests.ts

```diff
@@ -0,0 +1,46 @@
+import { generateHTMLPage } from 'sonarwhal/dist/tests/helpers/misc';
+import { RuleTest } from 'sonarwhal/dist/tests/helpers/rule-test-type';
+import * as ruleRunner from 'sonarwhal/dist/tests/helpers/rule-runner';
+import { getRuleName } from 'sonarwhal/dist/src/lib/utils/rule-helpers';
+
+const ruleName = getRuleName(__dirname);
+
+const testsForDefaults: Array<RuleTest> = [
+    {
+        name: `HTML and JS with P3P headers should fail`,
+        reports: [{ message: 'P3P is deprecated and should not be used' }, { message: 'P3P is deprecated and should not be used' }],
```

Comment:
Move `P3P is deprecated and should not be used` to a variable.

Reply:
Done